tutorials/debug/debugger_panel.rst

```diff
@@ -0,0 +1,105 @@
+.. _doc_debugger_panel:
+
+Debugger panel
+==============
+
+Many of Godot's debugging tools, including the debugger, can be found in the
+debugger panel at the bottom of the screen. Click on **Debugger** to open it.
+
+.. image:: img/overview_debugger.png
+
+The debugger panel is split into several tabs, each focusing on a specific task.
+
+Debugger
+++++++++
+
+The Debugger tab opens automatically when the GDScript compiler reaches
+a breakpoint in your code.
+
+It gives you a `stack trace <https://en.wikipedia.org/wiki/Stack_trace>`__,
+information about the state of the object, and buttons to control
+the program's execution.
+
+You can use the buttons in the top-right corner to:
+
+- Skip all breakpoints. That way, you can save breakpoints for future
+  debugging sessions.
+- Copy the current error message.
+- **Step Into** the code. This button takes you to the next line of code,
+  and if it's a function, it steps line-by-line through the function.
+- **Step Over** the code. This button goes to the next line of code,
+  but it doesn't step line-by-line through functions.
+- **Break**. This button pauses the game's execution.
+- **Continue**. This button resumes the game after a breakpoint or pause.
+
+Errors
+++++++
+
+This is where error and warning messages are printed while running the game.
+
+You can disable specific warnings in **Project Settings > Debug  > GDScript**
```

Comment:
Nitpick, but I'd prefer it to be fixed right away so I don't have to make a subsequent commit :slightly_smiling_face: 

```suggestion
You can disable specific warnings in **Project Settings > Debug > GDScript**.
```